source/source_stats_reporter.go

```diff
@@ -51,6 +57,7 @@ var (
 	responseTimeout        = tag.MustNewKey(metricskey.LabelResponseTimeout)
 )
 
+//ReportArgs interface
```

Comment:
This is a struct, not sure if we want to add this here.

Reply:
I can remove it.. it was just giving an suggestion to be exported or commented.. so I added it there